### DISCUSSION TITLE
Change typing speed / rate of "Write Text" field

### DISCUSSION THREAD

**Discussion by MikeLemo1** [streamdeck-linux-gui/streamdeck-linux-gui#217] [Q&A] [answered]
The typing speed / rate of the "Write Text" function is noticeable slow taking from 80-100ms per character to spit text 
which is undesirable for the long run ("Running tmux `:` command shortcuts with it so you want it to be as instant as possible") is it possible to edit it outside of the source code with the ui or do we have to be more intrusive? 
Is the former is impossible where should I start looking to do the latter?

**Comment by coolapso** [comment]
Hi! thank you for your feedbaack, just want to let you know that this is not something that is going to be fixed: https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/discussions/203 plese give it a try to [streamcontroller](https://github.com/StreamController/StreamController)

**Reply by MikeLemo1** [reply]
fine. and StreamDeck Doesn't install smoothly on ubuntu 24.04 according to the installation instructions...

**Reply by coolapso** [reply]
Can you provide any details about that? that's maybe related with the python version and that maybe should fix it

**Reply by MikeLemo1** [reply]
Well I just followed the official install doc with the latest pythos 3.12 venv and then with python3.11 venv but gives me an error related to xdp dependencies, posted an issue of streamcontroller repo

**Reply by coolapso** [reply]
Aaaw you talking about streamcontroller that doesn't install correctly? Have you tried Flatpak ? its probably the best option if you don't intend to do any development.

**Reply by MikeLemo1** [reply]
I think we talk about the same streamcontroller, anyways don't wanna install flatpak on ubuntu just for that shitty streamdeck just because there's no other option...
Why don't people just use apt to distribute packages?

**Reply by coolapso** [reply]
even thought I may agree with you .. I am not really going to debate on that, it will end nowhere. 

I am sure that streamcontroller mainteiners will accept your contributions if you;

* Open up a pull request adding the necessary steps to build, release and push such .deb package.
* Find a way to make such package reach the main deb / ubuntu repositories or provide and maintian the package repository

**Reply by MikeLemo1** [reply]
> even thought I may agree with you .. I am not really going to debate on that, it will end nowhere.
> 
> I am sure that streamcontroller mainteiners will accept your contributions if you;
> 
> * Open up a pull request adding the necessary steps to build, release and push such .deb package.
> * Find a way to make such package reach the main deb / ubuntu repositories or provide and maintian the package repository

Forgot to say which was relevant to my last replay that I did open an issue in the repo there still waitin....

**Reply by oparada1988** [reply, marked as answer]
> > even thought I may agree with you .. I am not really going to debate on that, it will end nowhere.
> > I am sure that streamcontroller mainteiners will accept your contributions if you;
> > 
> > * Open up a pull request adding the necessary steps to build, release and push such .deb package.
> > * Find a way to make such package reach the main deb / ubuntu repositories or provide and maintian the package repository
> 
> Forgot to say which was relevant to my last replay that I did open an issue in the repo there still waitin....

The project is pretty new, and its one person putting their time for us to have a better Steamdeck app alternative. I implore you to be more understanding and polite. Please remember Linux and Open Source works because talented people chose to give up their time to write/build applications
